buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -131,6 +133,7 @@ class ClusterFormationTasks {
     /** Adds a task to extract the elasticsearch distribution */
     static Task configureExtractTask(String name, Project project, Task setup, NodeInfo node) {
         List extractDependsOn = [project.configurations.elasticsearchDistro, setup]
+        // TODO assert that project.configurations.elasticsearchDistro.singleFile is from a project and not the cache
```

Comment:
It happened to me today - I ended up using an outdated rpm that wasn't wire compatible with the current one ant chasing phantoms for hours.  It'd make obvious errors that can be insidious otherwise.